src/app/service/feature.service.ts

```diff
@@ -12,4 +12,9 @@ export class FeatureService {
   public stratifier(): boolean {
     return (window as any).env.features.stratifier;
   }
+
+  // noinspection JSMethodCanBeStatic
+  public stratifierMinimalNumberBiobanks(): number {
```

Comment:
This is certainly no feature toggle. It's feature specific configuration. Let's discuss this later. Not in This PR.